docs/source/developer_guide/continuous_integration.rst

```diff
@@ -6,20 +6,28 @@ Continuous integration
 We use continuous integration services on GitHub to automatically check
 if new pull requests do not break anything and meet code quality
 standards such as a common `coding style <#Coding-style>`__.
+Before setting up Continuous Integration, be sure that you have set
```

Comment:
might be even clearer if you preface this by a "requires" or "preconditions" header.
Common thing in tech instruction manuals, and slightly reduces the risk that a reader skips it because they think it is part of the summary paragraph.

Reply:
on 2nd thought, that might overcomplicate things